po/zh_CN.po

```diff
@@ -132,153 +131,154 @@ msgstr "更改家区域密码需要认证。"
 msgid ""
 "Home of user %s is currently absent, please plug in the necessary storage "
 "device or backing file system."
-msgstr ""
+msgstr "用户 %s 的家目录当前不存在，请插入必要的存储设备或下层文件系统。"
 
 #: src/home/pam_systemd_home.c:292
 #, c-format
 msgid "Too frequent login attempts for user %s, try again later."
-msgstr ""
+msgstr "用户 %s 的登录尝试过于频繁，请稍后重试。"
 
 #: src/home/pam_systemd_home.c:304
 msgid "Password: "
-msgstr ""
+msgstr "密码： "
 
 #: src/home/pam_systemd_home.c:306
 #, c-format
 msgid "Password incorrect or not sufficient for authentication of user %s."
-msgstr ""
+msgstr "用户 %s 的密码不正确或不足以完成认证。"
 
 #: src/home/pam_systemd_home.c:307
 msgid "Sorry, try again: "
-msgstr ""
+msgstr "抱歉，请重试： "
 
 #: src/home/pam_systemd_home.c:329
 msgid "Recovery key: "
-msgstr ""
+msgstr "恢复密钥： "
 
 #: src/home/pam_systemd_home.c:331
 #, c-format
 msgid ""
 "Password/recovery key incorrect or not sufficient for authentication of user "
 "%s."
-msgstr ""
+msgstr "用户 %s 的密码/恢复密钥不正确或不足以完成认证。"
 
 #: src/home/pam_systemd_home.c:332
 msgid "Sorry, reenter recovery key: "
-msgstr ""
+msgstr "抱歉，请重新输入恢复密钥： "
 
 #: src/home/pam_systemd_home.c:352
 #, c-format
 msgid "Security token of user %s not inserted."
-msgstr ""
+msgstr "用户 %s 的安全令牌未插入。"
 
 #: src/home/pam_systemd_home.c:353 src/home/pam_systemd_home.c:356
 msgid "Try again with password: "
-msgstr ""
+msgstr "请尝试重新输入密码： "
 
 #: src/home/pam_systemd_home.c:355
 #, c-format
 msgid ""
 "Password incorrect or not sufficient, and configured security token of user "
 "%s not inserted."
-msgstr ""
+msgstr "密码不正确或无效，且用户 %s 配置的安全令牌未插入。"
 
 #: src/home/pam_systemd_home.c:376
 msgid "Security token PIN: "
-msgstr ""
+msgstr "安全令牌 PIN： "
 
 #: src/home/pam_systemd_home.c:393
 #, c-format
 msgid "Please authenticate physically on security token of user %s."
-msgstr ""
+msgstr "请在用户 %s 的安全令牌上进行物理认证。"
 
 #: src/home/pam_systemd_home.c:404
 #, c-format
 msgid "Please confirm presence on security token of user %s."
-msgstr ""
+msgstr "请确认用户 %s 的安全令牌存在。"
```

Comment:
Google translate says that this means "Please confirm that the security token for user %s exists.".

Reply:
Indeed this translation is somewhat misleading. I might not check carefully when reviewing other's and translating myself. After consulting the source code, I guess this part of code should express that asking user to check if the requested user's authentication info exists on the security token, instead of checking if the user have their security token.